lib/utils/sax-stream.js

```diff
@@ -1,14 +1,36 @@
-const {Writable} = require('stream');
-const saxes = require('saxes');
+const Stream = require('readable-stream');
+const {SaxesParser} = require('saxes');
 
-module.exports = class SAXStream extends Writable {
-  constructor() {
-    super();
-    this.sax = new saxes.SaxesParser();
+module.exports = class SAXStream extends Stream.Transform {
+  constructor(eventTypes) {
```

Comment:
I see that `eventTypes` everywhere has the same value, as `sax-stream.js` is an internal utility I think you should use default value here.

Reply:
Good catch - fixed in https://github.com/exceljs/exceljs/pull/1135